rskj-core/src/main/java/org/ethereum/rpc/exception/RskErrorResolver.java

```diff
@@ -24,22 +25,40 @@ public JsonError resolveError(Throwable t, Method method, List<JsonNode> argumen
         if(t instanceof  RskJsonRpcRequestException) {
             error =  new JsonError(((RskJsonRpcRequestException) t).getCode(), t.getMessage(), null);
         } else if (t instanceof InvalidFormatException) {
-            error = new JsonError(-32603, "Internal server error, probably due to invalid parameter type", null);
+            error = new JsonError(JsonRpcError.INTERNAL_ERROR, "Internal server error, probably due to invalid parameter type", null);
         } else if (t instanceof UnrecognizedPropertyException) {
-            String message = getExceptionMessage((UnrecognizedPropertyException) t);
-            logger.error("JsonRPC error {} for method {} with arguments {}", message, method, arguments);
-            error = new JsonError(-32603, message, null);
+            error = new JsonError(
```

Comment:
Maybe create a `JsonRpcInvalidParamsError` to match `JsonRpcInternalError`? Maybe it's pointless